strategy/dates.md

```diff
@@ -10,6 +10,16 @@ Our next scheduled event is planned to be a two-part session on APIs ([Topic: Cr
 
 ## Past Events
 
+### January 2020 (Part 1)
```

Comment:
Let's keep this consistent... Maybe add a "part 1" in the link of the slides.
```suggestion
### January 2020
```

Reply:
Fixed